### ISSUE TITLE
charts.fluxcd.io is down

### ISSUE THREAD

**Issue by balazs-pinter** [fluxcd/flux#2505]
**Describe the bug**
Trying to upgrade/install fluxcd via helm charts but it doesn't work.

**To Reproduce**
Steps to reproduce the behaviour:
0. helm repo add
helm repo add fluxcd https://charts.fluxcd.io

Error: Looks like "https://charts.fluxcd.io" is not a valid chart repository or cannot be reached: Get https://charts.fluxcd.io/index.yaml: dial tcp: lookup charts.fluxcd.io: no such host

**Expected behavior**
Helm repo added

**Comment by stefanprodan** [comment]
The fluxcd.io domain is down and we are looking into solving the DNS issue. As a workaround to install the Helm charts clone the repos and use the chart source:

```
git clone https://github.com/flux/flux
cd flux
helm upgrade -i flux ./chart/flux ....

git clone https://github.com/flux/helm-operator
cd helm-operator
helm upgrade -i helm-operator ./chart/helm-operator .....
```

**Comment by stefanprodan** [comment]
We are back! thanks @balazs-pinter for reporting

**Comment by simonfuhrer** [comment]
down again? curl: (6) Could not resolve host: charts.fluxcd.io

**Comment by Cohedrin** [comment]
Also seeing this unresolved. For some reason it resolves via cloudflare dns `1.1.1.1`, but not on any other dns provider.

**Comment by stefanprodan** [comment]
We have transferred the fluxcd.io domain to Linux Foundation, please bear with us while they solve the DNS issues. We have no control over it anymore but we've notified LF about the missing records.
